server/sonar-server/src/test/java/org/sonar/server/rule/RegisterRulesTest.java

```diff
@@ -456,12 +463,29 @@ public void remove_system_tags_when_plugin_does_not_provide_any() {
     assertThat(result.getSystemTags()).isEmpty();
   }
 
+  @Test
+  public void ignore_template_rules_if_organizations_are_enabled() {
+    organizationFlags.enable(dbTester.getSession());
+    execute(new RepositoryWithOneTemplateRule());
+
+    List<RuleDefinitionDto> rules = dbClient.ruleDao().selectAllDefinitions(dbTester.getSession());
+    assertThat(rules).hasSize(0);
+  }
+
+  @Test
+  public void log_ignored_template_rules_if_organizations_are_enabled() {
+    organizationFlags.enable(dbTester.getSession());
+    execute(new RepositoryWithOneTemplateRule());
```

Comment:
hum... that's not an argument "keep things simple" when leaving holes in coverage